TODO.md

```diff
@@ -68,3 +68,5 @@
   every read/write range command.
 
 * Gossip should be changed to send only protobufs.
+* persist Gossip and the kv.leaderCache to make sure that restarting
+  a node is harmless.
```

Comment:
There is a delta scheme between any two nodes, but it needs to be primed. Unfortunately, no way to know what a random new node might have to send. We could send it keys that we already have, but that's not implemented and nevertheless will still require non-trivial bandwidth.